addons/mail/static/src/models/chat_window_manager/chat_window_manager.js

```diff
@@ -161,11 +161,11 @@ function factory(dependencies) {
         }
 
         /**
-         * Shift provided chat window to the right on screen.
+         * Swap provided chat window with the next one.
```

Comment:
we already have a "swap" method, could it be easier to use it instead of doing the computation manually here?

Reply:
We could but it's not really optimal as we'll make two useless `findIndex` calls

Reply:
Ok, no change for this then.